client/components/checkout.js

```diff
@@ -33,14 +34,20 @@ class Checkout extends Component {
   render() {
     const {sumAmount} = this
     const totalAmount = sumAmount(this.props.cart.orderProducts)
+    const userId = this.props.cart.userId
+    const checkoutCart = this.props.checkoutCart
```

Comment:
I think instead of const checkoutCart we should call this something else to not confuse the 2 variables since it is already declared. maybe just call it cartCheckout.

Reply:
const cartCheckout = this.props.cartCheckout

Reply:
yes good idea. will change.